pkg/sdk/errors.go

```diff
@@ -48,3 +47,230 @@ func decodeDriverError(err error) error {
 
 	return err
 }
+
+const ghIssueBodyTemplate = `
+**Provider Version**
+
+The provider version you are using.
+
+**Terraform Version**
+
+The version of Terraform you were using when the bug was encountered.
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**Expected behavior**
+
+A clear and concise description of what you expected to happen.
+
+**Code samples and commands**
+
+Please add code examples and commands that were run to cause the problem.
+
+**Additional context**
+
+Add any other context about the problem here.
+
+<!-- 
+Please provide additional error messages if we missed any (see the errors below and compare it with your console output)
+-->
+
+Errors:
+%s
+`
+
+// NewTopLevelError wraps an error with "final" error message of sdk.
+// It should be placed in the highest place of call stack to catch as much error context as possible.
+// TODO Is it possible to call this function somewhere high in the callstack, so it always be top level ? If not maybe this would be overkill / too hard to use
+// TODO It's possible to wrap errors multiple times, this function will try to keep only the one with most error context.
+func NewTopLevelError(err error) error {
+	// TODO if called multiple times, unwrap lower level errors and wrap err to have more context
+
+	return fmt.Errorf(`
+Snowflake Terraform Provider error!
```

Comment:
in general, i prefer to keep all references to Terraform Provider out of the SDK. Is there any way this can be specified through the client config?

Reply:
Yeah, that's a good point keeping in mind that later we want to extract SDK as a separate project.

Reply:
Removed, so no reference to TF Provider is left